libcob/call.c

```diff
@@ -60,65 +60,82 @@ FILE *fmemopen (void *buf, size_t size, const char *mode);
 	https://wiki.musl-libc.org/functional-differences-from-glibc.html#Unloading_libraries
 */
 
+static cob_settings		*cobsetptr = NULL;
+
 #ifdef	_WIN32
+	#define WIN32_LEAN_AND_MEAN
+	#include <windows.h>
+	#include <io.h>	/* for access */
 
-#define WIN32_LEAN_AND_MEAN
-#include <windows.h>
-#include <io.h>	/* for access */
```

Comment:
We only use indents within conditional compilation very rarely- so please revert that part.